File: KantechGatewayApp/Jobs/ReactivateProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class ReactivateProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Infrastructure.Logger.Info($"[Reactivate] card={row.GetValueOrDefault("Card ID")} emp={row.GetValueOrDefault("employee ID")} -> reactivated")
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/ReactivateProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class ReactivateProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            Infrastructure.KantechJobApi.TriggerJobApi(_jobKey, row);
            Infrastructure.Logger.Info($"[Reactivate] card={row.GetValueOrDefault("Card ID")} emp={row.GetValueOrDefault("employee ID")} -> reactivated", _logSub);
        }
    }
}